components/team_sidebar/components/team_button.jsx

```diff
@@ -73,7 +78,13 @@ class TeamButton extends React.Component {
         });
 
         if (!teamClass) {
-            teamClass = unread ? 'unread' : '';
+            if (unread) {
+                teamClass = 'unread';
+            } else if (isNotSpecialButton) {
+                teamClass = '';
+            } else {
+                teamClass = 'special';
```

Comment:
Why are you adding a `special` class here? Seems like you're styling the default team button, but trying to avoid applying styles to the "Add Team" button. I'm thinking there may be a cleaner way to do this.